enricher/fabric8/src/main/java/io/fabric8/maven/enricher/fabric8/GrafanaLinkEnricher.java

```diff
@@ -61,28 +61,28 @@ private String findGrafanaLink() {
         String version = null;
 
         // TODO - use the docker names which may differ from project metadata!
-        if (Strings.isNullOrBlank(projectName)) {
+        if (StringUtils.isBlank(projectName)) {
             projectName = project.getArtifactId();
         }
-        if (Strings.isNullOrBlank(version)) {
+        if (StringUtils.isBlank(version)) {
             version = project.getVersion();
         }
 
-        if (Strings.isNotBlank(projectName)) {
+        if (StringUtils.isNotBlank(projectName)) {
             query += "&var-project=" + projectName;
         }
-        if (Strings.isNotBlank(version)) {
+        if (StringUtils.isNotBlank(version)) {
```

Comment:
This is not from this PR, but usually building a URL using string concatenation is a bad idea, and it is always a better approach to use a HttpBuilder where you can generate everything using a DSL approach. So the best approach is to check if any of the libraries used have one or implement one so it can be reused everywhere.